### PR TITLE
beta | Gidk 3bj75g | Aditi | env difference fixed

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/3bj75g

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts [hunk 1]
```diff
@@ -11,27 +11,28 @@ import { GstReconcileActions } from '../../../../actions/gst-reconcile/GstReconc
...
-            })),
-            state('out', style({
-                transform: 'translate3d(100%, 0, 0)'
-            })),
-            transition('in <=> out', animate('400ms ease-in-out')),
-        ])
-    ]
+	selector: 'reconcile',
+	templateUrl: './reconcilation.component.html',
+	styleUrls: ['reconcilation.component.scss'],
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts:16]: Please change to `./reconcilation.component.scss`

**Review by prateek-giddh — APPROVED** [review]

**Review by prateek-giddh — COMMENTED** [review]
Please check comment

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please check comment

File: apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts (hunk 1 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/gst/filing/tabs/reconcilation/reconcilation.component.ts]: done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**